docs/api.rst

```diff
@@ -406,7 +406,7 @@ URL is an immutable object, every operation described in the
 section generates a new *URL* instance.
 
 .. method:: URL.build(*, scheme, user, password, host, port, path, query, \
-                      query_string, fragment, strict=False)
+                      query_string, fragment, encoded=False)
```

Comment:
@asvetlov can this be somehow synched/linked from the current source code?

Reply:
I'm talking only about making doc generator look up correct signatures, not generate docs from docstrings. It would help to keep things in sync

Reply:
Would be nice but I don't know such tool

Reply:
There's sphinx extension for referring code from docs. It might be possible to abuse that or write your own based on that if none exists.